tests/unit/plugins/modules/test_ipa_pwpolicy.py

```diff
@@ -100,7 +100,12 @@ def test_add(self):
             'minlength': '16',
             'maxfailcount': '6',
             'failinterval': '60',
-            'lockouttime': '600'
+            'lockouttime': '600',
+            'gracelimit': 3,
+            'maxrepeat': 3,
+            'maxsequence': 3,
+            'dictcheck': True,
+            'usercheck': True
```

Comment:
If you add a trailing comma here the next time someone adds something the diff will look cleaner :)
```suggestion
            'usercheck': True,
```
(Same below.)